lib/OpenQA/WebAPI/GruJob.pm

```diff
@@ -0,0 +1,89 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package OpenQA::WebAPI::GruJob;
+use Mojo::Base 'Minion::Job';
+
+use OpenQA::Utils 'log_error';
+
+sub execute {
+    my $self = shift;
+
+    # TTL
```

Comment:
This comment serves no purpose. I usually start comments with a verb in imperative (like Git commit messages) to describe what the subsequent section does. A single word is quite meaningless.